xwiki-commons-core/xwiki-commons-xml/src/test/java/org/xwiki/xml/internal/html/DefaultHTMLCleanerTest.java

```diff
@@ -457,7 +456,9 @@ void cleanHTMLTagWithNamespace()
      * check, see <a href="https://github.com/cure53/DOMPurify/commit/e32ca248c0e9450fb182e52e978631cbd78f1123">commit
      * e32ca248c0 in DOMPurify</a>.
      */
+    // FIXME: Test not passing anymore, apparently the template is surviving now.
     @Test
+    @Disabled
```

Comment:
Does it mean the PR is not ready yet ? It's not marked as such.

Reply:
Yes I need @michitux input on this, I forgot to mention it here too.

Reply:
As I noted, I believe that removing `<template>` tags inside `<select>` could be security-relevant. Unfortunately, I don't have more information than what's inside the commit I've linked to. The [standard](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#technical_summary) is pretty clear that the only allowed content of `<select>` is the following:

> Zero or more [`<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option), [`<optgroup>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) or [`<hr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) elements.

Therefore, it is not correct to allow `<template>` elements inside `<select>`.

Reply:
Ok, thanks so you confirm we should keep it. I'll check how to handle this and if we report a bug to htmlcleaner for what look like a possible regression.

Reply:
It should be possible to add a workaround for this on our side.